Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -1401,6 +1401,16 @@ public async void Hide()
                         await Task.Delay(100);
                     LastQuerySelected = false;
                     break;
+                case LastQueryMode.ActionKeywordPreserved or LastQueryMode.ActionKeywordSelected:
+                    var newQuery = _lastQuery.ActionKeyword;
+                    if (!string.IsNullOrEmpty(newQuery))
+                        newQuery += " ";
+                    ChangeQueryText(newQuery);
+                    if (Settings.UseAnimation)
+                        await Task.Delay(100);
+                    if (Settings.LastQueryMode == LastQueryMode.ActionKeywordSelected)
+                        LastQuerySelected = false;
+                    break;
```

Comment:
I believe it can never be null. Initially it gets assigned an instance of Query in this ViewModel's constructor, after that it always contains the last query.